dataset_generation/wikisplit/test_transform_data_hi.py

```diff
@@ -0,0 +1,76 @@
+import os
+import shutil
```

Comment:
this doesn't seem to have any hindi specific data. so, can we rename it to test_transform_data.py?

Reply:
oh, right! really sorry about that.